flowzone.yml

```
@@ -1366,10 +1369,13 @@ jobs:
           comment="$(gh pr view ${{ github.event.pull_request.number }} --json comments \
             --jq '.[].[] | select(.author.login=="${{ needs.versioned_source.outputs.author }}").body')"
 
-          EOF="$(openssl rand -hex 16)"
-          echo "text<<$EOF" >> $GITHUB_OUTPUT
-          echo "${comment}" >> $GITHUB_OUTPUT
-          echo "$EOF" >> $GITHUB_OUTPUT
+          # send message only if a human processed the stubbed release notes, otherwise no-op
+          if ! [[ "$comment" =~ '[only keep the important and rephrase]' ]]; then
```

```suggestion
          if ! [[ "$comment" =~ 'only keep the important and rephrase' ]]; then
```
I think the inclusion of the `[...]` will match on any character in your string, when you only want to match the whole phrase.

I could be wrong, it should probably be tested for both pass and fail conditions.